anyway/infographics_utils.py

```diff
@@ -275,14 +276,28 @@ def get_most_severe_accidents(table_obj, filters, start_time, end_time, limit=10
         entities = (
             "longitude",
             "latitude",
-            "accident_severity_hebrew",
+            "accident_severity",
             "accident_timestamp",
-            "accident_type_hebrew",
+            "accident_type",
         )
         return get_most_severe_accidents_with_entities(
             table_obj, filters, entities, start_time, end_time, limit
         )
 
+    @staticmethod
+    def localize_items(request_params: RequestParams, items: Dict) -> Dict:
+        for item in items["data"]["items"]:
+            try:
+                item["accident_severity"] = _(
+                    english_accident_severity_dict[item["accident_severity"]]
+                )
+                item["accident_type"] = _(english_accident_type_dict[item["accident_type"]])
+            except Exception as e:
```

Comment:
Can you elaborate your meaning regarding too broad? It only contains a single statement.

Reply:
There is a hierarchy in the exception model(inheritance) and so some exception can catch wide range of errors, most of the time we want to catch only errors that we knows that can occur - so we can detect any bug in our code. 

This article(for Java/C#) is giving a code example that might help you:
https://vulncat.fortify.com/en/detail?id=desc.structural.java.poor_error_handling_overly_broad_catch

Reply:
The problem is not with the number of lines but with the marked section:
![image](https://user-images.githubusercontent.com/58947331/102009701-f0f9bf80-3d41-11eb-8989-47faf0be893c.png)

Reply:
My goal in this try - except is to prevent an error in the dictionary reference or in `gettext` to affect the calling function. Same behavior for all types of exceptions.
If I want to list all exceptions specifically, how do I know what exceptions  `gettext()` may throw?

Reply:
There are 4 options:
1. Read the documentations 
2. Search google
3. Read the source code if available 
3. Try to create an error and see what happens